bbc/simorgh
# Update Article Container to use Scenario Data

Article container now uses the relevant Scenario data. 
Cypress test updated to look at scenario-25
Cypress test updated to limit to 3 or fewer font files. 

* [x] Fixes https://github.com/bbc/simorgh/issues/430


**Testing instructions**
```npm run dev```
Visit http://localhost:7080/article/scenario-25
You should the main content of the article rendered. Headline, image, paragraphs, etc.
<img width="1437" alt="screen shot of article" src="https://user-images.githubusercontent.com/3028997/43582724-3f748dde-9655-11e8-9b0c-2c721d094d9c.png">
